docs/features/Logging.md

```diff
@@ -47,6 +47,9 @@ event:
 synchronization
 cronjob: The cronjob that triggered the call
 command: The command options and input that triggered the call, [based on](https://github.com/symfony/symfony/blob/6.2/src/Symfony/Bridge/Monolog/Processor/ConsoleCommandProcessor.php)
+headers: The headers of the request.
+
+> **Note:** That PHP will lowercase the incoming headers (for HTTP/1), or the HTTP client will lowercase the headers (for HTTP/2). According to RFC 7230, headers in HTTP/1 are case-insensitive, and according to RFC 7540, headers in HTTP/2 may only be lowercase.
```

Comment:
I'd remove the word That at the start of this sentence, and add something along the lines of:
'This means that the headers can only be found in lowercase in the logs.'